atomic_reactor/inner.py

```diff
@@ -282,8 +365,33 @@ def add_floating_images(self, images):
         for image in images:
             self.add_floating_image(image)
 
+    def add_unique_images(self, images: List[str]) -> None:
```

Comment:
do we really need both methods (image & images) ? we had now also some *_images methods, which weren't used at all

now you will use it in load but you can just as well loop there, or rename those methods to _...